### PR TITLE
Speaker data and gpu additions

### PR DESCRIPTION
**Pull request by BenjaminGalliot**

### CODE REVIEW

File: docs/wiki/elpis-dev-recipe.md [hunk 1]
```diff
@@ -19,27 +19,55 @@ git clone --depth=1 -b elpis https://github.com/persephone-tools/espnet
...
-    --entrypoint /bin/bash \
+    -v ~/elpis-sandbox/espnet/egs/elpis:/espnet/egs/elpis \
+    -v ~/elpis-sandbox/espnet/utils/:/espnet/utils/ \
+    --entrypoint /bin/zsh \
     coedl/elpis:latest
 ```
 
+### CUDA (beta, for ESPnet).
+
+If you have a CUDA-compatible GPU, you can have better performances (at least for model training). For this purpose, CUDA drivers must be installed on host machine (look if `nvidia-smi` works), and you can try the CUDA-specific Dockerfile (`cuda.Dockerfile` in root folder):
```
Comment (benfoley) [inline docs/wiki/elpis-dev-recipe.md:45]: Please change `cuda.Dockerfile` name to updated location `gpu/Dockerfile`.

File: docs/wiki/elpis-dev-recipe.md [hunk 2]
```diff
@@ -19,27 +19,57 @@ git clone --depth=1 -b elpis https://github.com/persephone-tools/espnet
...
-    --entrypoint /bin/bash \
+    -v ~/elpis-sandbox/espnet/egs/elpis:/espnet/egs/elpis \
+    -v ~/elpis-sandbox/espnet/utils/:/espnet/utils/ \
+    --entrypoint /bin/zsh \
     coedl/elpis:latest
 ```
 
+### CUDA (beta, for ESPnet).
+
+If you have a CUDA-compatible GPU, you can have better performances (at least for model training). For this purpose, CUDA drivers must be installed on host machine (look if `nvidia-smi` works), and you can try the CUDA-specific Dockerfile (`gpu/Dockerfile` in root folder):
```
Comment (nicklambourne) [inline docs/wiki/elpis-dev-recipe.md:45]: This first sentence is a little awkward, maybe something like this instead:
> If you have a CUDA-compatible GPU it is possible to achieve better performance by utilising the GPU in training your models.

File: docs/wiki/elpis-dev-recipe.md [hunk 3]
```diff
@@ -19,27 +19,57 @@ git clone --depth=1 -b elpis https://github.com/persephone-tools/espnet
...
+    coedl/elpis-cuda:latest
+```
+
+After, to be sure that CUDA mode for model training is enabled, check in the `elpis-sandbox/elpis/elpis/engines/espnet/objects/model.py` file that the command includes the `--ngpu x` (x is the number of GPUs) argument as:
+
+```
+p = run(f"cd {local_espnet_path}; ./run.sh --nj 1 --ngpu 1 &> {run_log_path}")
+```
+
+**Still, this feature is beta for the moment, do it if you know what you do or if you need higher performances.**
```
Comment (nicklambourne) [inline docs/wiki/elpis-dev-recipe.md:72]: > This feature is currently in a beta stage. Utilising the GPU for training is only currently recommended for those who know what they're doing or those particularly interested in achieving higher performance.

**Review by nicklambourne — APPROVED** [review]
I have made a couple of wording suggestions to the recipe, but it looks good to me.

**Pull request merged**